burgerqueen/src/components/BreackfastItem.jsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-shadow */
+/* eslint-disable react/prop-types */
+import React from 'react';
+import { Icon } from '@iconify/react';
+
+const BreackfastItem = (props) => {
+  const { item, addItem, removeItem } = props;
+  // aqui se renderiza el menu del desayuno
+  return (
+    <div>
+      <ul>
+        <li className="item-name">{item.name}</li>
+        <li className="item-price">$ {item.price}</li>
+        <button type="button" onClick={() => removeItem(item)}>
```

Comment:
```suggestion
        <button type="button" onClick={() => {removeItem(item)}}>
```
https://developer.mozilla.org/es/docs/Web/JavaScript/Reference/Functions/Arrow_functions